pkg/types/intoto/v0.0.2/intoto_v0_0_2_schema.json

```diff
@@ -93,7 +93,10 @@
                     ],
                     "readOnly": true
                 }
-            }
+            },
+            "required": [
```

Comment:
To confirm, this shouldn’t break any existing callers, correct? The envelope should have always been specified?

Reply:
no, the envelope is required on upload anyway so this is actually more accurate of intent.